hypothesis-python/src/hypothesis/strategies/_internal/strategies.py

```diff
@@ -475,20 +475,22 @@ def is_simple_data(value):
         return False
 
 
-class SampledFromStrategy(SearchStrategy):
+class SampledFromStrategy(SearchStrategy[Ex]):
     """A strategy which samples from a set of elements. This is essentially
     equivalent to using a OneOfStrategy over Just strategies but may be more
     efficient and convenient.
     """
 
     _MAX_FILTER_CALLS = 10_000
 
-    def __init__(self, elements, repr_=None, transformations=()):
+    def __init__(self, elements, repr_=None, transformations=(), shrink_towards=0):
```

Comment:
I think we can declare `shrink_towards` as a classvar rather than an init argument.